### ISSUE TITLE
1.0-RC1 cannot be resolved

### ISSUE THREAD

**Issue by viyang** [virtualdogbert/logback-groovy-config#2]
Is there any available release version can be used directly?

**Comment by virtualdogbert** [comment]
I haven't released any version of this yet to maven central. I got the Java service part of it working so that it picks up the config automatically, however, Grails Setup overrides it and I've been going back and forth with Puneet Behl from the Grails team when he or I had a little time, but we haven't resolved that yet.

**Comment by ceki** [comment]
@virtualdogbert There might be users who are interested in logback.groovy configuration but uninterested in Grails who might benefit from a release.

**Comment by virtualdogbert** [comment]
@ceki I understand that, but it wasn't just Grails it was really SpringBoot and I didn't want to release it before I had time to sit down and understand why it wasn't working, which I did today. So I'm going to do a little more testing and cleanup tomorrow and see if I can get it released.

If you care to know my configurator was being called and initialized, but after that Spring Boot loaded its own defaults because it didn't know I had initialized the logging. There's a check for its initialization here:
https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/logback/LogbackLoggingSystem.java#L122
